libc/test/src/math/RoundToIntegerTest.h

```diff
@@ -57,12 +57,13 @@ class RoundToIntegerTestTemplate
 
     ASSERT_EQ(func(input), expected);
 
+    // TODO: Handle the !expectError case. It used to expect
```

Comment:
Can the rest of the TODO's have links to a GitHub issue like this one so that it could be easy to search later?  Thanks,